pages/download.html

```diff
@@ -309,6 +309,15 @@ <h3>{% trans "Linux / Source Code" %}</h3>
   </div>
 </div>
 
+<div id="snapshots" class="docs-section container">
+  <h2><a class="headline-link" href="#snapshots">{% blocktrans %}Development Snapshots{% endblocktrans %}</a></h2>
+  <p>
+    {% blocktrans with source_link="https://github.com/mixxxdj/mixxx" windows_snapshot_link="https://downloads.mixxx.org/builds/appveyor/cmake_build/" macos_snapshot_link="https://downloads.mixxx.org/builds/travis/main/" ubuntu_snapshot_link="https://downloads.mixxx.org/builds/travis/main/" %}Want to test and experiment with the latest changes?
+    The best way to keep up with Mixxx development is to grab the <a href="{{ source_link }}">source code</a> and compile Mixxx yourself.
+    Additionally, we also provide development snapshots for <a href="{{ windows_snapshot_link }}">Windows</a>,  <a href="{{ macos_snapshot_link }}">macOS</a> and <a href="{{ ubuntu_snapshot_link }}">Ubuntu</a>.{% endblocktrans %}
```

Comment:
Some developers choose to use the main branch for performance. Every change is peer reviewed, however, it is more likely that new bugs will be found on the main branch than stable or beta releases. For most users, we recommend using the `<a href="#stable">stable</a>` version.

Reply:
462691a69062c1bdc5a2420198c28222b83e7bc9